### ISSUE TITLE
DB Net++预训练模型的问题

### ISSUE THREAD

**Issue by Abraham-2020** [PaddlePaddle/PaddleOCR#9738]
你好，我有一些疑问想请教一下，期待回复。  我在复现DB++算法的时候，骨干网络是ResNet50，不使用预训练模型 pretrained_model = None，模型训练过程评估指标都为0。我以为不使用预训练模型，随着模型的训练，ResNet50网络中的参数会慢慢的学习，不太明白为什么会为0，是还需要其他的配置吗？
当我把骨干网络换成MobileNetv3，且同样pretrained_model = None时，模型会慢慢训练，不会出现评估过程指标为0的情况。
因为我想替换或者修改骨干网络，所以不使用预训练模型，从头开始训练，想问我需要注意什么，该怎么实现呢？
谢谢你的时间！
训练时的log文件
[train (1).log](https://github.com/PaddlePaddle/PaddleOCR/files/11239500/train.1.log)

**Comment by Abraham-2020** [comment]
@LDOUBLEV 可以看一下这个问题吗？

**Comment by thsno02** [comment]
loss 太高了。在我的例子里，loss 在 3 以上验证集上的 hmean 都是 0，这是很正常的现象。有两种解决方案：

+ 让它继续训练下去，慢慢等；
+ 调整你的学习率和 batch size，让 loss 下降的速度变快。

**Comment by Abraham-2020** [comment]
嗯嗯，谢谢你的建议呀，Have a nice day☺！
